test/conversion_test.go

```diff
@@ -658,8 +658,8 @@ status:
 )
 
 // TestTaskCRDConversion first creates a v1beta1 Task CRD using v1beta1Clients and
-// requests it by v1Clients to compare with v1 if the conversion has been
-// correctly executed by the webhook. And then it creates the v1 Task CRD using v1Clients
+// requests it by v1Clients to compare with v1 if the conversion has been correctly
+// executed by the webhook for roundtrip. And then it creates the v1 Task CRD using v1Clients
```

Comment:
Just curious about this test, correct me if I'm wrong
it tests:
1) v1beta1->v1->v1beta1
2)v1->v1beta1->v1
I wonder if we need the 2) test case? Since it is covered in 1)? 🤔

Reply:
Thanks, good question:) I think this is necessary because for the roundTrip we would like to check if the conversion for results from the first step would lead to Error. 
eg. for the 2nd case, when we call ConvertFrom `v1` CRD ->  converted `v1beta1` CRD, then we test if the result could be ConvertTo-ed `v1` CRD

Also since the storedVersion is going to get swapped and we would want to avoid issues converting back and forth start from either version.

Reply:
That sg!